rest/data/commit_queue.go

```diff
@@ -456,13 +457,15 @@ func tryEnqueueItemForPR(ctx context.Context, sc Connector, projectRef *model.Pr
 
 // sendGitHubCommitQueueError updates the GitHub status and posts a comment
 // after an error has occurred related to the commit queue.
-func sendGitHubCommitQueueError(ctx context.Context, env evergreen.Environment, sc Connector, pr *github.PullRequest, userRepo UserRepoInfo, err error) error {
+func sendGitHubCommitQueueError(ctx context.Context, env evergreen.Environment, sc Connector, pr *github.PullRequest, userRepo UserRepoInfo, skipStatus bool, err error) error {
 	if err == nil {
 		return nil
 	}
 
 	catcher := grip.NewBasicCatcher()
-	catcher.Wrap(thirdparty.SendCommitQueueGithubStatus(ctx, env, pr, message.GithubStateFailure, err.Error(), ""), "sending GitHub status update")
+	if !skipStatus {
+		catcher.Wrap(thirdparty.SendCommitQueueGithubStatus(ctx, env, pr, message.GithubStateFailure, err.Error(), ""), "sending GitHub status update")
```

Comment:
Branch protection rules might apply even for projects that _aren't_ using the new commit queue. Could we just remove this entirely and only post the comment in all cases? The original ticket / PR only ever really mentioned sending a comment. (https://github.com/evergreen-ci/evergreen/pull/6499)

Reply:
removed!